src/codecs/webp/options.tsx

```diff
@@ -193,9 +186,8 @@ export default class WebPEncoderOptions extends Component<Props, {}> {
         </label>
         <label>
           Filter strength:
-          <input
+          <range-input
             name="filter_strength"
-            type="range"
             min="0"
             max="100"
             disabled={!!options.autofilter}
```

Comment:
This is broken - filed https://github.com/GoogleChromeLabs/squoosh/issues/142

Reply:
fixed.
<img width="216" alt="screen shot 2018-09-17 at 9 43 50 pm" src="https://user-images.githubusercontent.com/105127/45659484-cbc66a00-bac2-11e8-97f8-8030ded566e9.png">